### DISCUSSION TITLE
How can I execute a recursive deletion without confirmation, like a `rm -rf ...`?

### DISCUSSION THREAD

**Discussion by stephane-klein** [jarun/nnn#1607] [Q&A] [answered]
When I press on `x` on folder, I have this prompt:

```
rm -rf hovered? [Esc cancels]
```
I press `y`, next I need to confirm each file and folder 🙈

```
rm : descendre dans le répertoire '.../resources' ? y
rm : descendre dans le répertoire '.../resources/_gen' ? y
rm : supprimer '.../resources/_gen/images' du type répertoire ? y
rm : descendre dans le répertoire '.../resources/_gen/assets' ?
```

How can I execute a recursive deletion without confirmation, like a `rm -rf ...`?

**Comment by jarun** [comment, marked as answer]
If you press `y` it should use `rm -rf`. The `-f` is for force so you should not have to press `y` repeatedly. You are probably not pressing `y`.

**Reply by stephane-klein** [reply]
I understand my mistake: when I see:

```
rm -rf hovered? [Esc cancels]
```

I was typing `<enter>` and not `y`!
